cleaning_services/data/sale_order_template_line.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="sale_order_template_line_1" model="sale.order.template.line">
+    <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    <field name="name"><![CDATA[Services]]></field>
+    <field name="product_uom_qty" eval="False"/>
+    <field name="display_type">line_section</field>
+  </record>
+  <record id="sale_order_template_line_5" model="sale.order.template.line">
+    <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    <field name="sequence">11</field>
```

Comment:
You can remove all sequences of the file and add `auto_sequence="1"` at the top (see examples in the codebase)